proof/refine/ARM/Syscall_R.thy

```diff
@@ -1522,72 +1522,181 @@ crunch sch_act_sane[wp]: cteDeleteOne sch_act_sane
    simp: crunch_simps unless_def
    rule: sch_act_sane_lift)
 
+lemma getCapReg_corres_gen:
+  "corres (\<lambda>x y. x = to_bl y) cur_tcb cur_tcb'
+          (get_cap_reg rg) (getCapReg rg)"
```

Comment:
This and some of the following lemmas sort of feels like they should be earlier, but I guess if nothing else uses them then here is fine.